src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -13,6 +13,13 @@
      * числа 96, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        String s = text.replaceAll("[^-?[0-9]]", " ").trim();
```

Comment:
we can add:
```suggestion
        String s = text.replaceAll("[^-?[0-9]]+", " ").trim();
```
and then remove line 17